api/generated.go

```diff
@@ -85,9 +93,10 @@ type SignResponse struct {
 
 // VerifyRequest defines model for VerifyRequest.
 type VerifyRequest struct {
-	PlainText string    `json:"plainText"`
-	PublicKey PublicKey `json:"publicKey"`
-	Signature string    `json:"signature"`
+	Jwk       *JWK       `json:"jwk,omitempty"`
+	PlainText string     `json:"plainText"`
```

Comment:
shouldn't this (and signature, also) be of type []byte instead of string? Since encoding/decoding (from/to base64) is a concern of the JSON library?

Reply:
generated code, can't do anything about it other than change the generator to support `format: byte`

Reply:
ok